### DISCUSSION TITLE
Is it working?

### DISCUSSION THREAD

**Discussion by eletrofss** [3mora2/WPP_Whatsapp#10] [General]
i saw that playright wasn't passing the bot test on other sites, is it working with wpp now? what is the problem with threads breaking that you fixed?

**Comment by eletrofss** [comment]
Also, can we find the stuff that tries to check for bots and modify using an extension?

**Reply by 3mora2** [reply]
what you mean?

**Comment by 3mora2** [comment]
playwright not safe thread, you can't run two or more method in same time as sync, i think its relate to [greenlet](https://github.com/python-greenlet/greenlet).
I copy class safe thread of playwright from [minet](https://github.com/medialab/minet/blob/master/minet/browser/threadsafe_browser.py)
